platform/android/MapboxGLAndroidSDKTestApp/src/androidTest/java/com/mapbox/mapboxsdk/location/LocationComponentTest.kt

```diff
@@ -198,13 +224,19 @@ class LocationComponentTest : EspressoTest() {
     val componentAction = object : LocationComponentAction.OnPerformLocationComponentAction {
       override fun onLocationComponentAction(component: LocationComponent, mapboxMap: MapboxMap,
                                              style: Style, uiController: UiController, context: Context) {
-        component.activateLocationComponent(context,
-          style,
-          null,
-          LocationComponentOptions.builder(context)
-            .staleStateTimeout(200)
-            .enableStaleState(false)
-            .build())
+
+
+        locationComponentActivationOptions = LocationComponentActivationOptions
+                  .builder(context, style)
+                .locationEngine(null)
```

Comment:
This should be `.useDefaultLocationEngine(false)` instead.

Reply:
Ok, replaced `.locationEngine(null)` with `.useDefaultLocationEngine(false)`